src/go/rpk/pkg/api/admin/api_config.go

```diff
@@ -65,3 +60,90 @@ func (a *AdminAPI) SetLogLevel(name, level string, expirySeconds int) error {
 	path := fmt.Sprintf("/v1/config/log_level/%s?level=%s&expires=%d", url.PathEscape(name), level, expirySeconds)
 	return a.sendOne(http.MethodPut, path, nil, nil)
 }
+
+type ConfigPropertyItems struct {
+	Type string `json:"type"`
+}
+
+type ConfigPropertyMetadata struct {
+	Type         string              `json:"type"`
+	Description  string              `json:"description"`
+	Nullable     bool                `json:"nullable"`
+	NeedsRestart bool                `json:"needs_restart"`
+	Visibility   string              `json:"visibility"`
+	Units        string              `json:"units,omitempty"`
+	Example      string              `json:"example,omitempty"`
+	Items        ConfigPropertyItems `json:"items,omitempty"`
+}
+
+type ConfigSchema map[string]ConfigPropertyMetadata
+
+type ConfigSchemaResponse struct {
+	Properties ConfigSchema `json:"properties"`
+}
+
+func (a *AdminAPI) ClusterConfigSchema() (ConfigSchema, error) {
+	var rawResp []byte
+	err := a.sendOne(http.MethodGet, "/v1/cluster_config/schema", nil, &rawResp)
```

Comment:
For here and below:

`sendOne` errors if the admin API has more than one admin URL, and is meant to be used when you need to direct an API request to one specific node (setting the log level, for example). Are these meant to be `sendAny`? I don't see any specific `--host` flag in the commands below.

Reply:
Updated these to be sendAny